adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -1,5 +1,16 @@
 = sycl_khr_command_graph
 
+The graph abstraction defines an explicitly lazy execution model by decoupling
```

Comment:
Nit but given this comes immediately after the title I think `command graph` makes a bit more sense here since we are introducing the concept for the first time.

Maybe this needs a larger conversation about naming terminology consistency though.